mandatory/2-function-creation.js

```diff
@@ -16,7 +24,11 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(numb) {
+  return numb <= 100? `${(numb.toFixed(2))}%` :`100%`
+}
+
+console.log(formatPercentage(97.5555))
```

Comment:
Consider searching for console.log in the vscode before pushing changes to make sure you haven't forgotten to remove any logs